uach-retrofill.js

```diff
@@ -92,20 +92,24 @@ async function getUserAgentUsingClientHints(hints) {
     return Promise.resolve();
   }
 
-  // Verify that this is a Chromium
+  // Verify that this is a Chromium-based browser
   let is_chromium = false;
   let is_edge = false
   let chromium_version;
   navigator.userAgentData.brands.forEach(value => {
     if (value.brand == 'Chromium') {
-      is_chromium = true;
+      // Let's double check the UA string as well, so we don't accidentally
+      // capture a headless browser or friendly bot (which should report as
+      // HeadlessChrome or something entirely different).
+      is_chromium = true && /\sChrome/.test(navigator.userAgent);
```

Comment:
OK, in theory this is what I originally intended to do.

(But there's no way to really know)

Reply:
Can't the `true && ` just drop completely?

Reply:
Heh, yes. Thank you.